src/cockpit/packages.py

```diff
@@ -15,193 +15,247 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-
 import collections
+import contextlib
+import functools
 import gzip
-import hashlib
 import json
 import logging
 import mimetypes
 import os
 import re
 import shutil
 from pathlib import Path
-from typing import Callable, ClassVar, Dict, List, Optional, Pattern, Tuple
+from typing import Callable, ClassVar, Dict, Iterable, List, NamedTuple, Optional, Pattern, Tuple, TypeVar, Union
 
 from cockpit._vendor.systemd_ctypes import bus
 
 from . import config
+from ._version import __version__
 
-VERSION = '300'
```

Comment:
See above -- this could be `APIVERSION = 1`, or `301` if we really have to be monotonous to the previous value.

Reply:
Let's talk about this.  I'm not sure I understand what you mean.